src/clj/rems/api/services/resource.clj

```diff
@@ -6,6 +6,9 @@
             [rems.db.resource :as resource])
   (:import (org.postgresql.util PSQLException)))
 
+(defn get-resource [id] (resource/get-resource id))
+(defn get-resources [filters] (resource/get-resources filters))
```

Comment:
Wouldn't it make sense to refer these instead of having trivial wrapper code? Or just use with the ns?

Reply:
we're not just using them, we're re-exporting them so that users only need the services.resource ns